data-es6.js

```diff
@@ -13910,6 +13931,7 @@ exports.tests = [
       */},
       res: {
         xs6:         false,
+        chrome50:    true,
```

Comment:
@littledan looks like related this problem https://github.com/tc39/ecma262/pull/263